### PR TITLE
Upgraded jquery 2.2.4 to 3.3.1 and resolved deprecation notices

### PR DESCRIPTION
**Pull request by marcofranssen**
Upgraded jquery from 2.2.4 to 3.3.1.

As far I can see and tested on my own webblog there are no issues.

Resolved some deprecation notices using following script in main.js as well.
<script src="https://code.jquery.com/jquery-migrate-3.0.1.js"></script>

Please also do some sanity test if all plugins still work as I'm not using all of the plugins and this is a major version upgrade which might cause some trouble.

### CODE REVIEW

**Comment by ppoffice** [comment]
I think we should stay at jQuery 2.x for now in case some third party plugins breaks. We need a thorough test before moving on.

**Comment by marcofranssen** [comment]
Did we already do some test? On my personal blog I didn't see any issues so far https://marcofranssen.nl

I don't have all the plugins enabled though...

**Comment by marcofranssen** [comment]
I did rebase this branch on the master branch as well so you can have a straight forward merge ;)

File: source/js/main.js [hunk 1]
```diff
@@ -109,12 +109,12 @@
...
     autoHideMenus();
 
-    $(window).resize(function () {
+    $(window).on('resize', function () {
         autoHideMenus();
     });
 
     // Fold second-level menu
-    $('.main-nav-list-item').hover(function () {
+    $('.main-nav-list-item').on('hover', function () {
```
Comment (ppoffice) [inline source/js/main.js]: This breaks the main navbar dropdown.

Comment (marcofranssen) [reply source/js/main.js]: What exactly is broken? How do I reproduce?
https://marcofranssen.nl/ seems to be working fine.

Can you attach screenshot of the issue?

Comment (ppoffice) [reply source/js/main.js]: image
Dropdown menu no longer expands when hovered. I already fixed it.

Comment (marcofranssen) [reply source/js/main.js]: aaah. I don't have dropdowns on my side :( thanks for fixing. Sorry I missed it out.

**Pull request merged**